{{cookiecutter.project_name}}/.dockerignore

```diff
@@ -0,0 +1,16 @@
+.cache
+.coverage
+.coverage.*
+docker-compose.yml
```

Comment:
We should either also ignore `docker-compose.override.yml` or just go with `docker-compose*` to also catch potential ci.yml files.

Reply:
Good catch, yep indeed we should exclude all possible docker-compose files. Updated.